packages/strapi-plugin-content-type-builder/admin/src/containers/AttributesPickerModal/index.js

```diff
@@ -0,0 +1,182 @@
+/**
+ *
+ * AttributesPickerModal
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+// import { connect } from 'react-redux';
+// import { bindActionCreators, compose } from 'redux';
+import pluginId from '../../pluginId';
+
+import AttributeOption from '../../components/AttributeOption';
+import BodyModal from '../../components/BodyModal';
+import FooterModal from '../../components/FooterModal';
+import HeaderModal from '../../components/HeaderModal';
+import HeaderModalTitle from '../../components/HeaderModalTitle';
+import WrapperModal from '../../components/WrapperModal';
+
+import attributes from './attributes.json';
+
+class AttributesPickerModal extends React.Component {
+  // eslint-disable-line react/prefer-stateless-function
+  state = { isDisplayed: false, nodeToFocus: 0 };
+
+  componentDidMount() {
+    const { isOpen } = this.props;
+
+    if (isOpen) {
+      this.addEventListener();
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    const { isOpen } = this.props;
+
+    if (prevProps.isOpen !== isOpen) {
+      this.updateNodeToFocus(0);
+
+      if (isOpen) {
+        this.addEventListener();
+      } else {
+        this.removeEventListener();
+      }
+    }
+  }
+
+  getAttributes = () => {
+    const { plugins } = this.context;
+    const appPlugins = plugins.toJS ? plugins.toJS() : plugins;
+
+    return attributes.filter(attr => {
+      if (appPlugins.hasOwnProperty('upload')) {
+        return true;
+      }
+
+      return attr.type !== 'media';
+    });
+  };
+
+  addEventListener = () => {
+    document.addEventListener('keydown', this.handleKeyDown);
+  };
+
+  removeEventListener = () => {
+    document.removeEventListener('keydown', this.handleKeyDown);
+  };
+
+  handleClick = type => {
+    const { emitEvent } = this.context;
+    const { push } = this.props;
+
+    emitEvent('didSelectContentTypeFieldType', { type });
+    push({ search: `modalType=attributeForm&attributeType=${type}&settingType=base&actionType=create` });
+  };
+
+  /* istanbul ignore next */
```

Comment:
You look ignoring many lines, can you use a block ?

Reply:
It doesn't work... So I have to do it this way.